components/ActivityCardPreview/index.js

```diff
@@ -22,14 +21,19 @@ export default function ActivityCardPreview({ activity, onToggleBookmark }) {
           width={400}
```

Comment:
The layout prop in a next Image is legacy which means that it should not be used anymore. You can use it but it's not a good practice.